scipy-1.0/paper.tex

```diff
@@ -642,6 +642,35 @@ \subsection*{SciPy Matures}
 called PyData.
 
 
+\subsection{*Documentation}
+Early versions of SciPy had minimal documentation.  Travis
+Oliphant wrote \textit{A guide to NumPy}, an 8000 word e-book, which
+documented the underpinnings and API for compiled components, but did
+not provide reference documentation for the $\sim$2000 routines then in
+NumPy, nor said much about SciPy.  Realizing that the package had gotten too large for a single
+person to document, Joseph Harrington founded the SciPy Documentation
+Project in late Spring 2008, hiring Stefan van der Walt as documentation
+editor.  Pauli Virtanen, Emmanuelle Gouillart, and Gael Varoquaux wrote
+numpydoc, a system for collecting, reviewing, and safely integrating
+documentation with the Python sources.  The system, based on Sphinx,
+produced text, hypertext, and PDF-formatted documents from a single
+source document.  Harrington proposed and they established a
+documentation template and a review system.  Using the SciPy mailing
+list, they solicited community help, and by the end of the summer had
+high-quality documentation for the most frequently used NumPy and SciPy
+routines.  Each contributor of 1000 words or more received a T-shirt
+reading "SciPy Documentation Marathon: I WTFM", emphasizing that the
+project was no mere coding sprint.  In 2009, van der Walt became a core
+developer and David Goldsmith stepped up as lead editor.  The lead
+editor's job included maintaining discussion and interest on the mailing
+lists, which led to high participation.  By 2010, over 75 people had
+contributed significantly to over 1000 printed pages of reference
+documentation\cite{harrington2008scipy, harrington2009scipy}.
```

Comment:
Additional reference, for what it's worth: http://conference.scipy.org/proceedings/scipy2008/paper_5/full_text.pdf